### ISSUE TITLE
Variable name mayhem...

### ISSUE THREAD

**Issue by Redferne** [mycontroller-org/mycontroller#352]
The last few SNAPSHOT versions has changed the sensor variable legends...

Screenshot shows a set of MQTT sensors, MySensors and RFLink.
Those enormously long legend names including everything is just confusing and impossible to read.

Please make it possible to customize what should be displayed, both in dashboard, rooms details and basically everywhere for each sensor.

![sensor_names_wierd](https://cloud.githubusercontent.com/assets/2379821/22319600/85edd43a-e384-11e6-8fd4-958938377349.png)

**Comment by jkandasa** [comment]
@Redferne Changed this feature bit different. we have to enable ` Enable unique name` checkbox to get unique chart name. Otherwise looks as we had earlier.
![image](https://cloud.githubusercontent.com/assets/1004403/22343033/8f6a456e-e41c-11e6-8423-b1bbadfcf036.png)

**Comment by Redferne** [comment]
Nice!  Still believe that it should be possible to customize the display even further. See below screenshot. I've redmarked some areas available for improvement. The Title and Legend of each display should be customizable. It is probably easiest to support variables here in addition to the text string. The variables could be any of the column headings in the sensor list such as; $Network, $Gateway,  $Node, $EUI, $Sensor, $Name, $Type, $Variable, $Room,  $Lastseen. 

Looking at screenshot, Porchlights is single graph and the complete Variable Details should be removed since we already see Name and % Values in the graph. 

Goodnight buttons should also not display any Variable Details in this example. Intention is to turn off all lights in a group for bedtime. (How to send the on/off to a group from custom widget? The group button available is 'always' in the wrong state so I need to turn on all lights and then turn them off :-(  ) 

Outside Temperature: Here it would be prudent to maybe show $Room + $Lastseen as Varable Legend.

Voltages; Here is Enable unique name ticked and Legend is truncated and half invisible. The data is MQTT from a UPS and here it would nice to have $Node + " ( " + $EUI + " ) " in the Title and only $Name as Variable Legend.

If these parameters were customizable it would be so awesome. 

![20170126_233939](https://cloud.githubusercontent.com/assets/2379821/22355117/1cd80d3a-e429-11e6-900f-a90b661c526a.jpg)

**Comment by jkandasa** [comment]
@Redferne I guess this is somewhat duplicate request of https://github.com/mycontroller-org/mycontroller/issues/342

I will find a way to customise most of the things.

**Comment by Redferne** [comment]
Yes. Thank you for considering these improvements. There is so much potential in MyController!
